From 9d873c3b741fa2f38cdfa1086025d2ba2b2264ec Mon Sep 17 00:00:00 2001
From: Zhenbo Gao <zhenbo.gao@windriver.com>
Date: Mon, 3 Sep 2018 17:36:37 +0800
Subject: [PATCH 001/176] Revert "tipc: fix memory leak in
 tipc_accept_from_sock()"

This reverts commit 6d0519e75609652a7f3be097eae676fc94c8b41c.

A big patch set for upgrading TIPC(to v4.18) will come soon

Signed-off-by: Zhenbo Gao <zhenbo.gao@windriver.com>
---
 net/tipc/server.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/net/tipc/server.c b/net/tipc/server.c
index f4c1b18..3cd6402 100644
--- a/net/tipc/server.c
+++ b/net/tipc/server.c
@@ -313,7 +313,6 @@ static int tipc_accept_from_sock(struct tipc_conn *con)
 	newcon->usr_data = s->tipc_conn_new(newcon->conid);
 	if (!newcon->usr_data) {
 		sock_release(newsock);
-		conn_put(newcon);
 		return -ENOMEM;
 	}
 
-- 
1.7.9.5

